docs/monitoring/index.md

```diff
@@ -36,12 +36,13 @@ Follow along the [Quick Start](./quick-start) guide to learn how.
 
 App settings and actions include, but are not limited to:
 
-  - inviting others to your App
-  - [alert rules](../alerts)
   - changing your App's plan
-  - connecting Apps
-  - heartbeat alert cration
-  - App ownership transfer
+  - [managing alert rules](../alerts)
+  - [connecting Apps](https://sematext.com/docs/guide/integrations-guide/#connected-apps)
+  - [heartbeat alert creation](https://sematext.com/docs/alerts/creating-heartbeat-alerts/)
+  - [inviting others to your App](https://sematext.com/docs/team/app-guests/)
+  - [App ownership transfer](https://sematext.com/docs/team/transfer-apps/)
+  - [setting up a Sematext Monitoring Agent to ship metrics](https://sematext.com/docs/monitoring/quick-start/#setting-up-monitoring-agents)
   - scheduled report emails (aka Subscriptions)
```

Comment:
@cpipilas I think there is a ticket for this getting a page, too.

Reply:
Added subtask in the relevant ticket -> https://sematext.atlassian.net/browse/SC-14812